### PR TITLE
fix(mapi): sandbox patients document tables filled in

### PR DESCRIPTION
**Pull request by RamilGaripov**
refs. [metriport/metriport#821](https://github.com/metriport/metriport-internal/issues/821)

### Dependencies

- Downstream: [_[840]_](https://github.com/metriport/metriport-internal/pull/840)

### Description

- Added some logic to fill in generic information on sandbox patients' documents

### CODE REVIEW

**Review by leite08 — APPROVED** [review]

File: api/app/src/external/commonwell/document/document-query-sandbox.ts [hunk 1]
```diff
@@ -116,3 +132,63 @@ export async function sandboxGetDocRefsAndUpsert({
 
   return result;
 }
+
+function addSandboxFields(docRef: DocumentReference): DocumentReference {
+  if (docRef.content) {
+    const fileExt = docRef.content[0]?.attachment?.contentType?.split("/")[1]?.trim();
```
Comment (Goncharo) [inline api/app/src/external/commonwell/document/document-query-sandbox.ts:138]: we have a function for this https://github.com/metriport/metriport/blob/6aed56ac95875148ed1ea4aa78794608144572de/api/app/src/external/commonwell/document/shared.ts#L31-L36

File: api/app/src/external/commonwell/document/document-query-sandbox.ts [hunk 2]
```diff
@@ -116,3 +132,63 @@ export async function sandboxGetDocRefsAndUpsert({
...
+    } else if (fileExt === "pdf") {
+      (docRef.description = "Physical Examination"),
+        (docRef.type = {
+          coding: [
+            {
+              display: "Encounter",
+            },
+          ],
+        });
+    } else if (fileExt === "tif") {
```
Comment (Goncharo) [inline api/app/src/external/commonwell/document/document-query-sandbox.ts:171]: this should be `else if (fileExt === "tif" || fileExt === "tiff") {`

https://en.wikipedia.org/wiki/TIFF

**Review by Goncharo — CHANGES_REQUESTED** [review]

File: api/app/src/external/commonwell/document/document-query-sandbox.ts (hunk 1 above)
Comment (RamilGaripov) [reply api/app/src/external/commonwell/document/document-query-sandbox.ts:138]: Tried using it with both `attachment.title` and `attachment.contentType` - it returns an empty string for files of "tif" extension. See the image: https://imgur.com/f7PoIMF

Comment (RamilGaripov) [reply api/app/src/external/commonwell/document/document-query-sandbox.ts]: Resolved

**Review by Goncharo — APPROVED** [review]
:shipit:

**Pull request merged**